config.docker-template.php

```diff
@@ -10,7 +10,7 @@
 $CFG->dbname    = getenv('MOODLE_DOCKER_DBNAME');
 $CFG->dbuser    = getenv('MOODLE_DOCKER_DBUSER');
 $CFG->dbpass    = getenv('MOODLE_DOCKER_DBPASS');
-$CFG->prefix    = 'm_';
+$CFG->prefix    = 'mdl_';
```

Comment:
AFAIK this got committed by @T4m directly on master (with @OdyX and @astons-ra supervision :-P )

Reply:
Solved in another merge request, this one can be dismissed.